R/target_market_share.R

```diff
@@ -316,15 +315,72 @@ target_market_share <- function(data,
   data <- data %>%
     pivot_wider2()
 
+  data <- data %>%
+    left_join(green_or_brown, by = c("sector", "technology")) %>%
+    left_join(tmsr_or_smsp, by = "green_or_brown") %>%
```

Comment:
because these are objects defined above (not shown by default in this diff, it's hard to know if they should exist here... it would be easier to read/understand if you call the objects/functions directly here since they don't appear to be modified anywhere above (`r2dii.data::green_or_brown` and `tmsr_or_smsp()`)

Reply:
Ok I tried to be more explicit about these datasets. The thing is, they are used twice, and the functions are not terribly close together (they can't be really). So I'm not sure how to make it any more obvious where they come from, without importing them twice.